frontend/packages/vkt/src/interfaces/clerkExamEvent.ts

```diff
@@ -0,0 +1,61 @@
+import { Dayjs } from 'dayjs';
+
+import { EnrollmentStatus, ExamLanguage, ExamLevel } from 'enums/app';
+import { WithId, WithVersion } from 'interfaces/with';
+
+interface Person {
+  identityNumber: string;
+  lastName: string;
+  firstName: string;
+  email: string;
+  phoneNumber: string;
+  street?: string;
+  postalCode?: string;
+  town?: string;
+  country?: string;
```

Comment:
Keskusteltiin @terova kanssa että nämä tiedot siirtyvät varmaan ilmoittautumisen `ClerkExamEventEnrollment` alle. `ClerkEnrollment` vois muuten riittää hyvin tuon interfacen nimenä?

Reply:
Jees, päivitettään yhdessä backend linjan muutoksen kanssa. Nimesin nyt vain "Enrollment", tota Clerk:hän ei tosiaan tarvi viljellä joka paikkaan

Reply:
Joo, kentät `email` kentästä alaspäin siirtyvät tosiaan Enrollment-interfacen alle, kunhan tän mergaa pois: https://github.com/Opetushallitus/kieli-ja-kaantajatutkinnot/pull/336

Reply:
siirretty ->